src/api/users/controller.js

```diff
@@ -89,6 +115,7 @@ const getUserStats = async (req, res) => {
                 blog: githubData.data.blog,
             },
             stats,
```

Comment:
btw, what you implemented is already there in stats
<img width="368" alt="image" src="https://github.com/user-attachments/assets/db347cfa-fa10-4244-aadd-561f1db76ffa">
<img width="408" alt="image" src="https://github.com/user-attachments/assets/38f5e600-087c-4899-9019-215df5cbe06f">
remove that `repositoryBreakdown`